plugins/jvm-abi-gen/src/org/jetbrains/kotlin/jvm/abi/JvmAbiOutputExtension.kt

```diff
@@ -171,19 +154,22 @@ class JvmAbiOutputExtension(
                                 return abiMetadataProcessor(delegate)
                             }
 
-                            override fun visitEnd() {}
-                        }, 0)
+                            override fun visitEnd() {
+                                keptFields.sortedBy(FieldNode::name).forEach { it.accept(cv) }
+                                keptMethods.sortedBy(MethodNode::name).forEach { it.accept(cv) }
```

Comment:
```suggestion
                                keptFields.sortedWith(compareBy(FieldNode::name, FieldNode::signature)).forEach { it.accept(cv) }
                                keptMethods.sortedWith(compareBy(MethodNode::name, MethodNode::signature)).forEach { it.accept(cv) }
```
Similarly to the above, please make sure there's a test which would fail if we remove signature comparator